Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstanding.cs

```diff
@@ -47,6 +50,12 @@ public bool AllowSpatialUnderstanding
                 return true;
             }
         }
+
+        /// <summary>
+        /// Has any mesh sectors been scanned
+        /// </summary>
+        public bool HasMeshSectors { get { return UnderstandingCustomMesh.HasMeshSectors; } }
```

Comment:
previously it was private - so this was implemented. Merged! Will remove the property